command.go

```diff
@@ -20,7 +20,7 @@ type Command struct {
 	Description string
 	Usage       []string
 	RoleNeeded  *Role
-	Handler     func(caller *discordgo.Member, message *discordgo.Message, args []string) error
+	Handler     func(session *discordgo.Session, caller *discordgo.Member, message *discordgo.Message, args []string) error
```

Comment:
Why? you can access the session via the global variable `discord`. All this change is doing is making your otherwise small pr touch literally every command!

Reply:
didn't know it was a global